sass/components/_studylegend.scss

```diff
@@ -241,6 +249,11 @@
             background: themed('IndicatorInfoBg');
         }
 
+        &__content {
+            background: #ff0000;
```

Comment:
do we have to consider dark theme?

Reply:
@balakrishna-binary oops my bad, just fixed it.